domain/src/test/java/com/teamagam/gimelgimel/domain/dynamicLayers/remote/SendRemoteAddDynamicEntityRequestInteractorTest.java

```diff
@@ -50,15 +49,7 @@ public void whenExecuted_shouldCallRemoteSourceWithEntity() throws Exception {
     mTestSubject.execute();
 
     //Assert
-    verify(mRemoteSourceMock).addEntity(any(), eq(mGeoEntityMock));
+    verify(mRemoteSourceMock).addEntity(eq(mSingleDynamicLayer), eq(mGeoEntityMock));
```

Comment:
I used the captor because I needed to assert between 'DynamicLayerPresentation' and `DynamicLayer`.
This test actually assert equality by reference